profiles/base.yaml

```diff
@@ -91,6 +91,8 @@ combat_analyze_retry_count: 3
 manipulate_threshold:
 # manipulate only to train empathy, or always manipulate when above the threshold.
 manipulate_only_to_train: false
+# NPCs added here will be manipulated even if they are included on the ignore_npcs list
+manipulate_npcs:
```

Comment:
If this is an array, it should be in base-empty as well as `manipulate_npcs:  []`